index.html

```diff
@@ -2161,10 +2234,20 @@ <h3>Presentations</h3>
           </dd>
           <dt><var>proof</var></dt>
           <dd>
-The <a>verifiable presentation</a> MAY include a <code>proof</code>
-<a>property</a>. If present, the value SHOULD be used to express a securing
-mechanism, such as those listed in the
-Verifiable Credentials Specifications Directory [[?VC-SPECS]].
+            <p>
+                The <a>verifiable presentation</a> MAY include a <code>proof</code>
+                <a>property</a>, which refers to a separate <a>named graph</a> containing a single proof.
```

Comment:
my understanding is that multiple proofs are allowed here, and that in the case of BBS, the secure these information graphs in very different ways.

Reply:
@OR13 I have no opinion on this, and I would prefer if this was raised and discussed as a separate issue. The text in this version simply reused what is currently in the official draft.

Could you raise a separate issue and mark this conversation as resolved?